Src/SwisPowerShell/ConnectSwis.cs

```diff
@@ -178,7 +178,8 @@ private InfoServiceProxy ConnectNetTcp()
                 if (Streamed.IsPresent)
                 {
                     binding.TransferMode = TransferMode.Streamed;
-                    binding.PortSharingEnabled = true;
+                    // TODO - is this a problem?
+                    //binding.PortSharingEnabled = true;
```

Comment:
It doesn't seem like this matters, since SWIS is always using net.tcp port sharing and this client works fine without `binding.PortSharingEnabled = true`. We should just remove the comments here.

Reply:
Done.